packages/flutter/lib/src/widgets/visibility.dart

```diff
@@ -224,39 +224,46 @@ class Visibility extends StatelessWidget {
   /// objects, to be immediately created if [visible] is true).
   final bool maintainInteractivity;
 
+  static VisibilityMarker of(BuildContext context) {
```

Comment:
Wondering: Should this just return true/false? Do you have any plans to extend VisibilityMarker? Or ideas what we should add in the future?

Reply:
Good point - returning a boolean is cleaner.